identityClient/identityClient.go

```diff
@@ -664,6 +666,20 @@ func (c *E3dbIdentityClient) IdentityLogin(ctx context.Context, realmName string
 	return identity, err
 }
 
+// InternalCreateIdentityLoginAudit creates an audit of the login attempt returning all the information about the login
+// attempt and error (if any)
+func (c *E3dbIdentityClient) InternalCreateIdentityLoginAudit(ctx context.Context, params InternalIdentityLoginAudit) (*InternalIdentityLoginAudit, error) {
+	var loginAudit *InternalIdentityLoginAudit
+	path := c.Host + internalIdentityServiceBasePath + fmt.Sprintf("/%s/%s", auditResourceName, loginResourceName)
+	// might want to update this so that it takes a part of the params part ...
```

Comment:
If you dont want an item in the struct to be sent over to the endpoint dont include the `json:___" then it wont be marshalled!